application.coffee

```diff
@@ -0,0 +1,28 @@
+pack = Packages.register
+  name: "application-enhancements"
```

Comment:
not sure if this is the best name for this package? could not think of anything better and I didn't think it was a good idea to name it `application` as that already exists in core